app/Http/Controllers/Profiles/ProfilesController.php

```diff
@@ -2,19 +2,18 @@
 
 namespace App\Http\Controllers\Profiles;
 
+use Illuminate\Support\Facades\Auth;
 use Inertia\Inertia;
 
 class ProfilesController
 {
     public function index()
     {
-        // We'll need to grab the mentor and mentee profiles (if they exist)
-        // for the currently authenticated user.  We'll then return these to the
-        // view.
+        $u = Auth::user();
 
         return Inertia::render('Profiles/Index', [
```

Comment:
I think the `Profiles/Index.tsx` page has been deleted so I'm bit confused about this controller does any more.. should we get rid of this controller?

Reply:
Yes :)